llvm/lib/Analysis/BasicAliasAnalysis.cpp

```diff
@@ -80,7 +84,7 @@ STATISTIC(SearchTimes, "Number of times a GEP is decomposed");
 
 // The max limit of the search depth in DecomposeGEPExpression() and
 // getUnderlyingObject().
-static const unsigned MaxLookupSearchDepth = 6;
+static const unsigned MaxLookupSearchDepth = 10;
```

Comment:
I think you won't get this upstreamed. Make it a command-line option and pass it from the driver?

Reply:
Done!